android-components/components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt

```diff
@@ -156,6 +156,13 @@ interface OAuthAccount : AutoCloseable {
      */
     suspend fun getTokenServerEndpointURL(): String?
 
+    /**
+     * Fetches the URL for the user to manage their account
+     *
+     * @return The URL which should be opened in a browser tab.
+     */
+    suspend fun getManageAccountURL(entryPoint: String): String
```

Comment:
```suggestion
    suspend fun getManageAccountURL(entryPoint: FxAEntryPoint): String
```

Is this `entryPoint` similar to the other `FenixFxAEntryPoint` implementation values? We perhaps want our API to use that instead. This should give a simple compile-time check for other users of this API to apply an explicit and known entry point value when using the account manager in different parts of the application.